### PR TITLE
First pass on moving from ResizeObserver to Container Queries

### LINKED ISSUE #1018: [Refactor] Use container queries for the Mavo toolbar
There’s a bit of code in `src/ui.bar.js` to adapt the Mavo toolbar to different sizing conditions. We should simply use container queries for it, since these days they’re [supported everywhere](https://caniuse.com/css-container-queries) (Baseline 2023!).

This may be a good issue for someone who has been looking for an excuse to try container queries 😄

### PR DESCRIPTION
**Pull request by DmitrySharabin**
Closes #1018.

https://github.com/mavoweb/mavo/assets/9166277/7700dacb-9c37-441c-9ccc-88d2757f8958

### CODE REVIEW

File: src/ui.bar.js [hunk 1]
```diff
@@ -100,57 +92,9 @@ let _ = Mavo.UI.Bar = class Bar {
...
-			this.resize();
-
-			if (self.ResizeObserver) {
-				this.resizeObserver = Mavo.observeResize(this.element, entries => {
-					this.resize();
-				});
-			}
-		}
-
 		Mavo.observers.resume();
```
Comment (DmitrySharabin) [inline src/ui.bar.js:95]: Do we still need it here?

File: src/ui.bar.js
Comment (DmitrySharabin) [inline src/ui.bar.js:1]: I wonder, do we still need these two lines? Do we still have observers that need to be paused and resumed here? 🤔

(line 12)
```js
Mavo.observers.pause();
```
(line 95)
```js
Mavo.observers.resume();
```

File: src-css/_bar.scss [hunk 2]
```diff
@@ -175,20 +177,25 @@
 	}
 
 	// Displaying mv-bar in small spaces
-	&.mv-compact {
-		.mv-status > span {
-			display: none;
+	@container mv-bar (width <= 40em) {
```
Comment (DmitrySharabin) [inline src-css/_bar.scss:180]: `40em` and `25em` are magic numbers obtained from experiments that “feel OK” in general cases. However, it's not the silver bullet for all the cases. If the Mavo bar has many buttons, these numbers need to be adjusted.

Comment (LeaVerou) [reply src-css/_bar.scss:180]: One thing at a time. It's ok to keep the magic numbers for now. Eventually we want to move to a component that handles its own layout.

File: src/ui.bar.js (hunk 1 above)
Comment (LeaVerou) [reply src/ui.bar.js:95]: I have no idea. Let's leave it for now, can't hurt.

**Review by LeaVerou — APPROVED** [review]
LGTM. That felt good, right? 😁

**Comment by DmitrySharabin** [comment]
> LGTM. That felt good, right? 😁

Yeah. Fells like magic. I’m glad I could work on it. We live in the future now. 😄

**Pull request merged**